src/main/resources/application.conf

```diff
@@ -5,7 +5,8 @@ akka {
 
 spray.can {
   server {
-    request-timeout = 30s
+    request-timeout = 180s
```

Comment:
3 minutes might be too high, but in some cases with 25 workspaces being created within 15 seconds, it was hitting a little over 2 minutes for them to complete.